cdap-ui/app/directives/macro-widget-toggle/macro-widget-toggle.html

```diff
@@ -97,11 +99,16 @@
         }"
       >
         <!-- If its macro enabled show a textbox -->
-        <input
+        <text-box
+          id="macro-input-{{MacroWidget.field.name}}"
+          value="MacroWidget.node.plugin.properties[MacroWidget.field.name]"
+          on-change="MacroWidget.onChange"
+        ></text-box>
+        <!-- <input
```

Comment:
nit: Can we remove the commented out code. We can always go back in git history if we need that snippet.